### PR TITLE
Pretty printing and syntactic sugar for mod

### PR DESCRIPTION
**Pull request by xwvvvvwx**
1. Adds a little more sugar to `act` for bitwise `AND` and `modInt` operations.
2. Cleans up pretty printing of `modInt` to bring it inline with the new `act` syntax
3. Removes pretty printing of `%Int`. This is basically never used within the `evm-semantics` and it's inclusion within a spec is probably an error. I removed it to avoid the possibility of a copy paste error from the debug view into a spec.

### CODE REVIEW

File: lib/kast.js [hunk 1]
```diff
@@ -293,8 +293,7 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "_-Int_"             : " - "   ,
       "_*Int_"             : " * "   ,
       "_/Int_"             : " / "   ,
-      "_%Int_"             : " % "   ,
-      "_modInt_"           : " % "   ,
+      "_modInt_"           : " mod " ,
```
Comment (asymmetric) [inline lib/kast.js]: I like `%` more than `mod`, but no strong feelings.

Comment (xwvvvvwx) [reply lib/kast.js]: `%` looks nicer, but I think it's lying a bit. `K` has both a `%Int` and a `modInt` with slightly different semantics (see [here](https://github.com/kframework/k/blob/master/k-distribution/include/builtin/domains.k#L379) and [here](https://github.com/kframework/kore/blob/master/docs/hooks.md#inttmod)). `modInt` is the one that is used in the `evm-semantics`, and I think it's good if the `klab` debugger faithfully reflects that.

Comment (ehildenb) [reply lib/kast.js]: They mean different things, btw. `%Int` and `modInt` in K have different implementations. `-5 modInt 3 =/= -5 %Int 3`. One gives back `-2`, the other `1`. I can't remember which is which, but KEVM specifically chooses the right ones.

**Comment by xwvvvvwx** [comment]
CI passing here: https://buildbot.dapp.ci/#/builders/2/builds/481

**Pull request merged**